draft-ietf-quic-manageability.md

```diff
@@ -809,20 +816,161 @@ off the connection ID may therefore cause undetectable and unrecoverable loss
 of state in the middle of a connection. Use of connection ID specifically
 discouraged for NAT applications.
 
-## Passive Network Performance Measurement and Troubleshooting
+## Network Address Translation (NAT)
```

Comment:
This section is too long for me.  I started reviewing it, but I lost the thread.  I don't think that this is an improvement.

Reply:
I actually agree. I was just trying to keep all of the points in Martin Duke's original text (while removing redundancy). PR #272 is a proposal for a minimum text version. Maybe we can start from there and re-add things if needed.